packages/manager/modules/cloud-universe-components/src/cui/grid.less

```diff
@@ -1,4 +1,5 @@
 @cui-grid-margin: 0.9375rem;
+@screen-sm-min: 801px;
```

Comment:
i'm just curious, why do we set screen-sm-min to 801px? is this legacy code ?

Reply:
yes, it comes from https://github.com/ovh/manager/blob/master/packages/manager/apps/cloud/client/app/app.less#L43